src/main/java/umk/mat/pajda/ProjektZespolowy/misc/ReviewConverter.java

```diff
@@ -30,9 +31,11 @@ public ReviewDTO createDTO(Review review) {
 
   public Review createEntity(ReviewDTO reviewDTO) {
     Review review = new Review();
+    review.setId(reviewDTO.getId());
```

Comment:
Is this related to POST, right? We don't need to set the ID because the database generates it automatically, and the frontend only sends to backend ID when editing a specific review.